src/components/AdminDashboard/CompleteIncident.js

```diff
@@ -139,20 +139,30 @@ const CompleteIncident = props => {
 
             {/* Tags */}
             <div className="dropdown-text-wrap">
-              <p className="complete-incident-dropdown-titles-bold">Tags</p>
-              <div>
-                {incident.tags ? incident.tags.join(", ") : ''}
-              </div>
+              <p className="complete-incident-dropdown-titles-bold">Tags:</p>
+              <div>{incident.tags ? incident.tags.join(', ') : ''}</div>
+            </div>
+
+            {/* Confidence Rating: */}
+            <div className="dropdown-text-wrap">
+              <p className="complete-incident-dropdown-titles-bold">
+                Confidence Rating:
+              </p>
+              <p>{(incident.confidence * 100).toFixed(2)}%</p>
             </div>
 
             {/* Edit button */}
-            <Button id="dropdown-edit-button" className="approve-reject-select" onClick={toggleEditor}>
+            <Button
+              id="dropdown-edit-button"
+              className="approve-reject-select"
+              onClick={toggleEditor}
+            >
               Edit
             </Button>
           </>
-        }
+        )}
 
-        {editing &&
+        {editing && (
```

Comment:
Your addition of a paren here would be a consistent formatting change, since you did that to the other one too.